Lib/glyphsLib/builder.py

```diff
@@ -463,6 +463,16 @@ def set_custom_params(ufo, parsed=None, data=None, misc_keys=(), non_info=()):
         if name in ('openTypeOS2WeightClass', 'openTypeOS2WidthClass'):
             value = int(value)
 
+        if name == 'Replace Feature':
+            if len(value.split(';')) > 2:
```

Comment:
why requiring two semicolons instead of one? The Glyphs Handbook says 

>  The first four letters must be the feature name (such as ‘liga’), followed by a semicolon and the new feature code.

I guess you did that to ignore replacing with an empty or incomplete feature code (every statement must end with a semicolon in FEA).

Maybe we should log a warning saying something like "the custom parameter value is not properly formatted" when we hit the `else` block instead of silently ignoring the custom parameter?

Reply:
Yes, it would definitely be good to do a little more validation here since the value could be any kind of junk.